src/Uno.UI/UI/Xaml/Controls/Layouter/Layouter.cs

```diff
@@ -269,6 +269,8 @@ protected Size MeasureChild(View view, Size slotSize)
 			var frameworkElement = view as IFrameworkElement;
 			var ret = default(Size);
 
+			// NaN values are accepted as input for MeasureOverride, but are treated as Infinity.
+			slotSize = slotSize.NumberOrDefault(MaxSize);
```

Comment:
Maybe this change should be done ins `Measure()` too... no?

Reply:
It was, in #2098